### PR TITLE
LanguageTest.py: Add tests for Languages class

### PR DESCRIPTION
**Pull request by MariosPanag**
Add tests in coalib.bearlib.languages.Language.py
for Languages class.

Closes https://github.com/coala/coala/issues/4037

**For short term contributors:** we understand that getting your commits well
defined like we require is a hard task and takes some learning. If you
look to help without wanting to contribute long term there's no need
for you to learn this. Just drop us a message and we'll take care of brushing
up your stuff for merge!

### Checklist

- [ ] I read the [commit guidelines](http://coala.io/commit) and I've followed
      them.
- [ ] I ran coala over my code locally. (*All commits have to pass
      individually.* It is not sufficient to have "fixup commits" on your PR,
      our bot will still report the issues for the previous commit.) You will
      likely receive a lot of bot comments and build failures if coala does not
      pass on every single commit!

After you submit your pull request, **DO NOT click the 'Update Branch' button.**
When asked for a rebase, consult [coala.io/rebase](https://coala.io/rebase)
instead.

Please consider helping us by reviewing other peoples pull requests as well:

- pick up any PR at <https://coala.io/review>
- review it (check <https://coala.io/reviewing> for more info)
- if you are sure that it needs work, use `cobot mark wip <URL>` to get it out
  of the review queue.

The more you review, the more your score will grow at coala.io and we will
review your PRs faster!

### CODE REVIEW

File: tests/bearlib/languages/LanguageTest.py [hunk 1]
```diff
@@ -27,3 +27,19 @@ def test_invalid_attribute(self):
...
+class LanguagesTest(unittest.TestCase):
+
+    def setUp(self):
+        self.items = Languages([Language.Python == 3])
+        self.tuple = (Language['Python 3.3, 3.4, 3.5, 3.6'],)
+
+    def tearDown(self):
+        pass
+
+    def test_language_tuple(self):
```
Comment (userzimmermann) [inline tests/bearlib/languages/LanguageTest.py]: You don't need to include the term 'language_' in every method name, since `Languages` is already in the class name. Or in other words: Please remove ;)

File: tests/bearlib/languages/LanguageTest.py [hunk 2]
```diff
@@ -27,3 +27,19 @@ def test_invalid_attribute(self):
...
+        self.assertTupleEqual(Languages(self.items), self.tuple)
+
+    def test_language_contains(self):
+        self.assertTrue('Python 2.7, 3.5' in Languages([Language.Python()]))
```
Comment (userzimmermann) [inline tests/bearlib/languages/LanguageTest.py]: Please also add tests with more than one language in a `Languages` tuple, and some negative tests.

**Comment by MariosPanag** [comment]
I tried to apply all your recommendations @userzimmermann, except ```add tests with more than one language in a Languages tuple```, because it's not possible for now as i explained in this issue https://github.com/coala/coala/issues/4048

**Comment by userzimmermann** [comment]
Thanks @MariosPanag ! Looks good so far :) But we definitely have to rethink the `Language.__eq__` implementation according to your https://github.com/coala/coala/issues/4048 before finishing this PR. So I'm setting this PR to WIP for now.

@sils Remember our `Language.__eq__`? :D It starts to cause trouble...

**Comment by MariosPanag** [comment]
Okey :) Lets discuss https://github.com/coala/coala/issues/4048 and after we fix it i will amend this commit to include tuples with more than one element.

**Comment by supergr35** [comment]
This should be marked blocked if https://github.com/coala/coala/issues/4048 should be fixed first.
